lib/manageiq/appliance_console/database_replication.rb

```diff
@@ -75,6 +75,10 @@ def generate_cluster_name
         say("Failed to get primary region number #{e.message}")
         logger.error("Failed to get primary region number #{e.message}")
         return false
+      rescue PG::UndefinedTable
```

Comment:
Any reason not to rescue this with the `PG::ConnectionBad` error above?  Is there something in the message that you don't want to display?

Reply:
I think it looks more friendly to user to communicate error in known terms (`no region exists`) than in some internal terms (`relation "miq_databases_id_seq" does not exist`)

Reply:
@bdunne In this particular BZ, appliance_console will raise `PG::UndefinedTable` instead of `PG::ConnectionBad` because ap tries to `SELECT last_value FROM miq_databases_id_seq` but `miq_databases_id_seq` doesn't exist (because of improper settings, regions not created). So here I rescue also for this error

Reply:
The `rescue` on https://github.com/ManageIQ/manageiq-appliance_console/pull/13/files#diff-96034652e522899e72c0060781012f58L74 seems to be that we can't even connect to the database.  This one is that we can connect but the database hasn't been regioned yet.  Is that correct?  Do the two situations have different resolutions?